security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,81 @@
+//OpenShift Compliance Operator Release Notes
+[id="compliance-operator-release-notes"]
+= Compliance Operator release notes
+:context: compliance-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+The Compliance Operator lets {product-title} administrators describe the desired compliance state of a cluster and provides them with an overview of gaps and ways to remediate them.
```

Comment:
```suggestion
The Compliance Operator lets {product-title} administrators describe the required compliance state of a cluster and provides them with an overview of gaps and ways to remediate them.
```

"Desire" is one of the words that we can't use because it doesn't translate well.

Reply:
Done